LandSandBoat/server
# [core] Add interrupts for job abilities (pet job abilities)

**_I affirm:_**

## What does this pull request do?

Adds the ability to interrupt job abilities, such as Wyvern Breaths

## Steps to test these changes

Use Restoring Breath, have the target run out of range, see "Too far away" message and the wyvern stops animating.
